vue-acme-store/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM node:10-alpine as build-deps
```

Comment:
do you feel that the setup you did for the frontend is enough to run the app on multiple environments (e.g. staging and production) or would you need to change something for it to work?